kola/harness.go

```diff
@@ -278,9 +278,15 @@ func RunTests(pattern, pltfrm, outputDir string) error {
 		torcxManifestFile.Close()
 	}
 
+	flight, err := NewFlight(pltfrm)
+	if err != nil {
+		plog.Fatalf("Flight failed: %v", err)
```

Comment:
nit: This is `"new flight: %v"` some places. Maybe standardize unless there's a reason not to.

Reply:
There's a pretty good mix of message formats within `cmd/kola`, and I used the local convention in each case.  I'll leave for now.